core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -66,6 +68,71 @@ class ReplicaManagerTest {
     metrics.close()
   }
 
+  @Test
+  def testAlterReplicaLogDirs() {
```

Comment:
It would be useful to add some high level comments on what the test does.

Reply:
Sure. The following comments are added:

```
// Verify that alterReplicaLogDirs() works while ReplicaAlterLogDirsThread is replacing current replica with future replica
// This is to simulate a corner case in which request handler threads tries to stop current replica movement
// as part of ReplicaManager.alterReplicaLogDirs() while the ReplicaAlterLogDirsThread is replacing
// current replica with the future replica
```